pkg/controller/servicebindingrequest/sbrcontroller.go

```diff
@@ -32,6 +32,9 @@ var (
 	// defaultPredicate default predicate functions
 	defaultPredicate = predicate.Funcs{
 		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectNew.GetObjectKind().GroupVersionKind().Kind == "secret" {
```

Comment:
@DhritiShikhar the `Kind` might actually be `Secret`(with capital `S`) so I'd suggest to use the one string comparison ignoring case, so:
```go
if strings.EqualFold(e.ObjectNew.GetObjectKind().GroupVersionKind().Kind, "secret") {
   return true
}
```

Reply:
like this https://github.com/DhritiShikhar/service-binding-operator/pull/1 :)